src/index.tsx

```diff
@@ -46,26 +46,34 @@ function Updaters() {
   )
 }
 
+const ReduxWrapper = () => {
+  const walletOverride = useAppSelector((state) => state.user.walletOverride)
```

Comment:
Back out this change. Get walletOverride/connectorOverride in the separate Web3Provider component (see overall comments).